src/doc/tarpl/lifetime-misc.md

```diff
@@ -0,0 +1,229 @@
+% misc
+
+This is just a dumping ground while I work out what to do with this stuff
+
+
+# PhantomData
```

Comment:
Perhaps keep this in the variance section? (And link to it from the drop section)